modules/hcp-get-ocp-channel.adoc

```diff
@@ -2,134 +2,82 @@
+. Check recommended {product-title} channels by running the following command:
++
+[source,terminal]
+----
+$ oc -n <hosted_cluster_namespace> get -o jsonpath='{.status.version.desired.channels}' hostedcluster<hosted_cluster_name>
+----
 
-See the following example of the `HostedCluster` CR that contains the channel configuration:
+. Update the `spec.release` field of the `HostedCluster` to trigger the control plane rollout by running the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> --type=merge -p '{"spec":{"release":{"image":"quay.io/openshift-release-dev/ocp-release@sha256:<release_payload>"}}}' <1>
+----
+<1> Replace `<release_payload>` with the {product-title} release payload that you want to use.
 
-[source,yaml]
-----
-spec:
-  autoscaling: {}
-  channel: stable-4.y <1>
-  clusterID: d6d42268-7dff-4d37-92cf-691bd2d42f41
-  configuration: {}
-  controllerAvailabilityPolicy: SingleReplica
-  dns:
-    baseDomain: dev11.red-chesterfield.com
-    privateZoneID: Z0180092I0DQRKL55LN0
-    publicZoneID: Z00206462VG6ZP0H2QLWK
-----
-<1> Replace `<4.y>` with the {product-title} release version you specified in `spec.release`. For example, if you set the `spec.release` to `ocp-release:4.16.4-multi`, you must set `spec.channel` to `stable-4.16`.
+. Set a channel in your hosted cluster that matches with your desired channel semantics. For more information about channels and releases, see "Understanding update channels and releases". Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> --type=merge -p '{"spec":{"channel":"stable-<4.y>"}}' <1>
+----
+<1> Replace `<4.y>` with the {product-title} release version you specified in `spec.release`. For example, if you set the `spec.release` to `ocp-release:4.18.4-multi`, you must set `spec.channel` to `stable-4.18`.
+
+. Check the list of available updates for your hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> -o jsonpath='{.status.version.availableUpdates}'
+----
 
-After you configure the channel in the `HostedCluster` CR, to view the output of the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields, run the following command:
+. Monitor the progress of your ongoing update:
 
+.. Monitor the status of the update rollout in the `HostedCluster` resource. You can check the progression of the update, timestamps and issues by running the following command:
```

Comment:
I think we require the oxford command: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas 

(before _and_) 

```suggestion
.. Monitor the status of the update rollout in the `HostedCluster` resource. You can check the progression of the update, timestamps, and issues by running the following command:
```